src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
There should be only **one line which separates class declaration and its method** which follows right after class declaration. (it is not a very strict rule, but a code is cleaner then)
```suggestion

    public static String getRandomColor() {
```